controllers/users.js

```diff
@@ -602,6 +601,31 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeUserNickname = async (req, res) => {
```

Comment:
**Suggestion**: I think the function name `updateDiscordNickname` will be more readable and easy to understand. Let me know what do you think about it